azure-pipelines.yml

```diff
@@ -60,17 +60,17 @@ stages:
               rustup_toolchain: stable
               target: 'x86_64-apple-darwin'
             linux-stable:
-              imageName: 'ubuntu-20.04'
+              imageName: 'rust:alpine'
               rustup_toolchain: stable
-              target: 'x86_64-unknown-linux-gnu'
+              target: 'x86_64-unknown-linux-musl'
```

Comment:
that would break all the package managers, if you want to add musl, it needs to be another target, not replace the existing one.

Reply:
Why would it? All the distros compile it themselves, IMO the only use case for a pre-compiled binary is downloading it to some arbitrary system, which we can't make any assumptions about. So it seems to me that for Linux we should ship only static binaries.

Reply:
> All the distros compile it themselves, IMO the only use case for a pre-compiled binary is downloading it to some arbitrary system, which we can't make any assumptions about

I would rather have both options than only one

> So it seems to me that for Linux we should ship only static binaries.

Last time I checked musl, performance was significantly worse. It might have been fixed since then but my results were kinda similar to https://andygrove.io/2020/05/why-musl-extremely-slow/

Reply:
Thanks for pointing this out, I didn't know. I'll close this PR then.